CS342MemberViews/Sources/ShriyaReddy/HomeButton.swift

```diff
@@ -0,0 +1,53 @@
+//
+// File.swift
+// Created by Shriya Reddy on 1/24/23.
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import MapKit
+import SwiftUI
+
+
+public struct HomeButton: View {
+    @State private var showDetails = false
+    
+    @State private var region = MKCoordinateRegion(
+            center: CLLocationCoordinate2D(latitude: 44.182205, longitude: -84.506836),
+            span: MKCoordinateSpan(latitudeDelta: 10, longitudeDelta: 10)
+        )
```

Comment:
Please only indent arguments one level lower than their declaration:
```suggestion
        center: CLLocationCoordinate2D(latitude: 44.182205, longitude: -84.506836),
        span: MKCoordinateSpan(latitudeDelta: 10, longitudeDelta: 10)
    )
```